Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.13.5
```

Comment:
this is very old, can we use a newer alpine? latest is 3.16.2

Reply:
probably the github template needs to be updated. I didn't look at it, will update thanks <3